test/projectManager.ts

```diff
@@ -166,6 +166,36 @@ export class ProjectManager {
             return ProjectManager.runPlatform(testRunDirectory, targetPlatform, true, targetEmulator);
         }
     }
+    
+    /**
+     * Navigates away from the application and then navigates back to it.
+     */
+    public static resumeApplication(delayBeforeResumingMs: number, targetPlatform: platform.IPlatform, namespace: string, testRunDirectory: string, targetEmulator: string): Q.Promise<void> {
+        var emulatorManager = targetPlatform.getOptionalEmulatorManager();
```

Comment:
This class is intended to be platform agnostic. Most of its functionality relies on the Cordova CLI. Can you add a `resumeApplication` method to the `IEmulatorManager` interface and implement if for both Android and iOS there, in `platform.ts`? I would create an `EmulatorManager` for Android as well (currently the android platform doesn't have one) and move this code here. The iOS specific code can go in `IOSEmulatorManager ` and the Android specific in a new `AndroidEmulatorManager` class.